templates/applications/common/cf/fargate-taskdef-base-properties.yml

```diff
@@ -0,0 +1,8 @@
+Family: !Join ['', [!Ref ProjectName, '-', !Ref EnvName, '-', !Ref AppName]]
```

Comment:
Should we make this file in `common` folder? Looks like although this setting applies to both backend app and LBfargate app, it only applies to fargate service. Once we have EC2 service then this setting does not apply.

Reply:
I think we'll have a bunch of fargate app types so I thought it'd be nice to move these to a common file. I named the file "fargate-*" to denote that this is fargate specific, does that make sense?